.gitignore

```diff
@@ -16,3 +16,6 @@ target/
 
 # results from HTTP API testing tool
 *run-*
+
+# Intelij Idea files
+.idea/*
```

Comment:
A `.gitignore` in a repo should normally only contain files being created by the repository itself, ie, build artifacts or local caches.

For IDE metadata the convention is to keep them in your [global ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).  

I'd appreciate if you could remove `.idea/` from `.gitignore` again and move it to your own global ignore file.